en/3-meeds.md

```diff
@@ -94,20 +95,20 @@ which leads to the following holding time for each class of holder :
 
 ## 3.6. Meeds DAO governance
 
-Meeds token holders need to stake tokens in order to vote on the Meeds DAO decisions such as managing the minting budget or electing DAO board members (annual mandate).
+xMEED token holders can freely participate in the governance of the Meeds DAO by voting with their tokens on decisions such as managing the budget allocation or electing DAO board members.
 
 ## 3.7. Decentralized finance (DeFi) user services
 
 The WoM will introduce the blockchain world to many people who do not use it today.
 
-Indeed, any work (whether digital or not) can be gamified and any engagement - rewarded with Meeds. Meeds are rendered available to users through an easy to use, blockchain based, wallet that “abstracts” any complexity of the crypto world to an average user.
+Indeed, Meeds can gamify and set a monetary value on any work (whether digital or not) and reward it with MEED tokens. Thanks to an easy-to-use, blockchain-based wallet app that simplifies access to the crypto world for an average user, rewards are tangible.
```

Comment:
I think that the use of the word "contribution" instead of "work" makes it more user-friendly and "Thanks" is an informal team not suitable in an official document.